docs/user/Dependencies.md

```diff
@@ -4,55 +4,53 @@ If your environment requires validated support for a specific version or range o
 
 The following list details our dependencies and versions (~ indicates multiple possible sources):
 
-SOURCE | NAME | VERSION
---- | --- | ---
-~ | python | 3.x
-~ | pip | 3.x
-~ | unzip | any
-~ | tar | any
-~ | docker | any
-~ | git | any
-~ | kustomize | 3.7.0
-~ | kubectl | 1.21 - 1.23
-~ | AWS IAM Authenticator | 1.18.9/2020-11-02
```

Comment:
Why was the authentication removed? Again, no docs explaining this

_Originally posted by @thpang in https://github.com/sassoftware/viya4-deployment/pull/224#discussion_r939031445_

Reply:
@thpang See https://github.com/sassoftware/viya4-deployment/commit/7e89b16edc7369683422b1950dd57a261be320b3
Removed by Norm on Apr 1, 2021, the doc was just outdated.